gnd/src/main/java/com/google/android/gnd/ui/projectselector/ProjectSelectorViewModel.java

```diff
@@ -15,48 +15,89 @@
  */
 package com.google.android.gnd.ui.projectselector;
 
+import android.util.Log;
+
 import androidx.lifecycle.LiveData;
 import androidx.lifecycle.MutableLiveData;
+
 import com.google.android.gnd.repository.DataRepository;
 import com.google.android.gnd.repository.Resource;
+import com.google.android.gnd.rx.Result;
 import com.google.android.gnd.system.AuthenticationManager;
 import com.google.android.gnd.ui.common.AbstractViewModel;
 import com.google.android.gnd.vo.Project;
-import io.reactivex.Completable;
+
+import io.reactivex.Observable;
+import io.reactivex.subjects.PublishSubject;
+
 import java.util.Collections;
 import java.util.List;
+
 import javax.inject.Inject;
 
 public class ProjectSelectorViewModel extends AbstractViewModel {
   private static final String TAG = ProjectSelectorViewModel.class.getSimpleName();
 
   private final DataRepository dataRepository;
   private final MutableLiveData<Resource<List<Project>>> projectSummaries;
-  private final AuthenticationManager authManager;
+  private final PublishSubject<Integer> projectSelections;
+  public final MutableLiveData<Project> activeProject;
+  public final MutableLiveData<Throwable> activateProjectErrors;
```

Comment:
I think I need to eat my words about making these public final. Since they're mutable, any client can accidentally update them. We really should expose them as `LiveData` via methods like `projectActivationLiveData()` and `activeProjectLiveData()`. Sorry that I overlooked this!

Reply:
Cool, I've changed these to private and added accessors. I've also updated the fragment appropriately.

I stuck with the existing naming pattern for live data accessors `get<data>()`. We can take on a future effort to rename these with the LiveData suffix, if you want.